package.json

```
@@ -9,6 +9,9 @@
     "start:server": "wait-on dist/server.js && node dist/server.js",
     "start": "npm run clean && npm-run-all --print-label --parallel start:*",
     "build": "npm run clean  && cross-env NODE_ENV=production && webpack --mode production",
+    "watch:server": "nodemon \"./dist/server.js\" --watch \"./dist\"",
```

🧐
По идее, так тоже должно работать
```suggestion
    "watch:server": "nodemon ./dist/server.js --watch ./dist",
```

---

Да, логично, спасибо